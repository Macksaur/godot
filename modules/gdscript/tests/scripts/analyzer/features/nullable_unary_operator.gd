func test():
    var ok := 0
    var foo: int? = null
    if foo == null:
        print("it's null!")
        ok += 1

    foo = 123
    if foo != null:
        print(-foo)
        ok += 1

    if ok == 2:
        print("Ok")
