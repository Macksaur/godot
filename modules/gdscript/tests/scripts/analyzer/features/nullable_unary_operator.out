GDTEST_OK
it's null!
-123
Ok
